layout/mainmenu_settings.xml

```diff
@@ -4,6 +4,7 @@
 	</styles>
 
 	<scripts>
+		<include src="file://{resources}/scripts/common/settings.js" />
 		<include src="file://{resources}/scripts/settings_shared.js" />
```

Comment:
Should settings_shared stuff be moved to common/settings.js ?

Reply:
Mmh no, settings_shared is actually a bunch of unique functionality for different settings pages, the structure is a carryover from CS:GO's settings. I'd quite like to refactor that but too much work for little gain currently IMO.